components/formats-gpl/src/loci/formats/in/BaseZeissReader.java

```diff
@@ -999,15 +1013,22 @@ else if (key.startsWith("Comments")) {
           imageDescription = value;
         }
         else if (key.startsWith("Acquisition Date")) {
-          if (timepoint > 0) {
-            timestamps.put(timepoint - 1, value);
-            addGlobalMetaList("Timestamp", value);
-          }
-          else {
-            timestamps.put(timepoint, value);
+          acquisitionDate = new Timestamp(DateTools.convertDate(parseTimestamp(value), DateTools.UNIX, DateTools.ISO8601_FORMAT_MS));
+        }
+        else if (image >= 0 && key.startsWith("Camera Acquisition Time")) { // Note Double variant for TIFF XML.
+          String date = DateTools.convertDate(parseTimestamp(value), DateTools.UNIX, DateTools.ISO8601_FORMAT_MS);
+          addSeriesMetaList(key, date);
+          if (timepoint != 0) { // First timepoint is duplicated for some reason.
+            timestamps.put(timepoint - 1, date);
+          } else {
+            timestamps.put(timepoint, date);
           }
           timepoint++;
         }
+        else if (image >= 0 && key.startsWith("ImageRelativeTime")) {
+          Double reltime = Double.parseDouble(value) * 1000 * 60*60*24;
```

Comment:
I'm not sure I follow why this value needs to be multiplied by the number of milliseconds per day, but perhaps a ```Time``` object could be used instead so that the units are obvious?